etc/pip-requires-client

```diff
@@ -1,18 +1,8 @@
 # All dependencies needed to run rucio client should be defined here
 setuptools>=36.8.0,<45.0.0; python_version < '3.0'          # Python packaging utilities (36.8.0 last py2.6 compatible version, 45.0 only Py3 compatible)
 setuptools>=45.0.0,<=50.3.2; python_version >= '3.0'        # Python packaging utilities
-argcomplete>=1.9.0,<=1.12.2                                 # Bash tab completion for argparse
 requests>=2.20.0,<=2.25.0                                   # Python HTTP for Humans.
 urllib3>=1.24.2,<1.26                                       # HTTP library with thread-safe connection pooling, file post, etc.
 dogpile.cache>=0.6.5,<0.7.0; python_version < '3.0'         # Caching API plugins (0.7.0 only Py3 compatible)
 dogpile.cache>=0.6.5,<=1.1.1; python_version >= '3.0'       # Caching API plugins
-boto>=2.49.0,<2.50.0                                        # S3 boto protocol
-tabulate>=0.8.0,<0.9.0                                      # Pretty-print tabular data
```

Comment:
These should not be removed here, but moved to optional dependencies in the client's setup.py

Reply:
I have moved these to optional dependencies. I am open to suggestions for the names of them, and also whether it would be better to merge some of the minor ones into a single extra that people could install if they don't care about the number of dependencies and want all the original functionality, rather than having several different ones.

Reply:
I think to put these as optional is the way to go. No need to include a specific protocol per default, if you are not using it.